tools/python/example_operator_perf_test.py

```diff
@@ -0,0 +1,141 @@
+"""
+Example python code for creating a model with a single operator and performance testing it with various
+input combinations.
+"""
+
+import onnx
+from onnx import helper
+from onnx import TensorProto
+from onnx import shape_inference
+import numpy as np
+import time
+import timeit
+import sys
+import onnxruntime as rt
+
+# if you copy this script elsewhere you may need to add the tools\python dir to the sys.path for this
+# import to work.
+# e.g. sys.path.append(r'<path to onnxruntime source>\tools\python')
+import ort_test_dir_utils
+
+# make input deterministic
+np.random.seed(123)
+
+
+#
+# Example code to create a model with just the operator to test. Adjust as necessary for what you want to test.
+#
+def create_model(model_name):
```

Comment:
The function is specific to topk, I would include it in the name.

Reply:
The intention is to provide an example implementation that can be used as a base for different ops with this file being cut-and-pasted. Because of that I wanted the file name and functions in it to be generic.